### PR TITLE
refactor: Apply lint rule `cred-class-field-display-name-miscased`

### PR DESCRIPTION
**Pull request by ivov**
https://github.com/ivov/eslint-plugin-n8n-nodes-base/blob/master/docs/rules/cred-class-field-display-name-miscased.md

### CODE REVIEW

**Comment by netroy** [comment]
instead of making an exception for `E-goi`, should we change the rule to enfore the name that you get after removing certain characters like `-`?

**Comment by ivov** [comment]
[`titleCase`](https://github.com/blakeembrey/change-case/blob/master/packages/title-case/src/index.ts) in the plugin capitalizes all words except articles and prepositions and conjunctions, whether hyphenated or not. This matches [traditional rules](https://www.proofreadnow.com/blog/bid/94159/Hyphenation-in-Titles) for casing of hyphenated nouns and adjectives in titles, which makes me think hyphenated brand names in nodes-base (rare as they are, no other example) are likely to follow the pattern, making `E-goi` an exception, but hard for me to say. Maybe @StarfallProjects knows better.

**Comment by StarfallProjects** [comment]
I think Ivan's approach will be right in most cases. The site he links describes a common style convention for title case headings (I think it may be drawn from the Chicago Manual of Style? Don't quote me on that though). 

Conventions vary depending on what style guide you follow. Are we basing our UI copy styles on any particular style guide? Worth noting that a lot of popular tech style guides (i.e. guides focused on digital rather than print, including Google's and Microsoft's) avoid title case altogether, but that's a separate issue: main thing is we're consistent.

**Comment by ivov** [comment]
> Are we basing our UI copy styles on any particular style guide?

Mostly David's indications on top of Jan's preexisting choices. David is not a fan of title case either but we kept it because it was so prevalent.

File: packages/nodes-base/package.json [hunk 1]
```diff
@@ -749,7 +749,7 @@
     "@types/tmp": "^0.2.0",
     "@types/uuid": "^8.3.2",
     "@types/xml2js": "^0.4.3",
-    "eslint-plugin-n8n-nodes-base": "^1.11.0",
+    "eslint-plugin-n8n-nodes-base": "^1.11.1",
```
Comment (netroy) [inline packages/nodes-base/package.json:752]: this shouldn't be making 16k changes in the lock file. something seems off.

Comment (ivov) [reply packages/nodes-base/package.json:752]: Installing with `^1.11.0` was pulling `1.11.0` despite `1.11.1` being released. I'll recheck.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.205.0`

**Pull request merged**